spec/helpers/helpers_spec.rb

```diff
@@ -1,17 +1,17 @@
 module Helpers
   def login(email, password)
     visit root_path
-    click_link "Войти"
+    click_link I18n.t('.layouts.application.login')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.